web/server/routers/account.ts

```diff
@@ -143,4 +144,40 @@ export const accountRouter = router({
         }
       }
     }),
+  get: protectedProcedure
+    .input(
+      z.object({
+        email: z.string().email(),
+        uid: z.string(),
+        name: z.string(),
+      })
+    )
+    .output(
+      z.object({
+        found: z.boolean(),
+      })
+    )
+    .query(async ({ ctx, input }) => {
+      const email = input.email;
+      if (email === null) {
+        throw new TRPCError({
+          code: "UNAUTHORIZED",
+          message: "Not authenticated",
+        });
+      } else {
+        const account = await findAccount(email);
+        if (account === null) {
+          return { found: false };
```

Comment:
throw tRPC error instead of returning a successful response since the user that's requesting to log in is not authorized